### PR TITLE
[MAB] MAB Production Behavior

### PR DESCRIPTION
**Pull request by mukundv-chrome**

### CODE REVIEW

File: configs/test/bigquery/datasets.yaml [hunk 1]
```diff
@@ -209,25 +209,19 @@ resources:
...
           - name: weight
             type: FLOAT
-  - name: fuzz-strategy-experiments-table
+  - name: fuzz-strategy-probability-table
     type: gcp-types/bigquery-v2:tables
     properties:
       datasetId: $(ref.main-dataset.datasetReference.datasetId)
       tableReference:
-        tableId: fuzz_strategy_experiments
+        tableId: fuzz_strategy_probability
```
Comment (mbarbella-chromium) [inline configs/test/bigquery/datasets.yaml]: Happen to know if this table still exists in the internal project from before? We'll likely need to delete it if so.

File: src/appengine/handlers/cron/fuzz_strategy_selection.py [hunk 2]
```diff
@@ -31,9 +31,7 @@
 from libs import handler
 from metrics import logs
 
-HIGH_TEMPERATURE_PARAMETER = .5
-MEDIUM_TEMPERATURE_PARAMETER = .25
-LOW_TEMPERATURE_PARAMETER = .15
+TEMPERATURE_PARAMETER = .15
```
Comment (mbarbella-chromium) [inline src/appengine/handlers/cron/fuzz_strategy_selection.py]: Could you leave a comment explaining why we went with this temperature?

File: configs/test/bigquery/datasets.yaml (hunk 1 above)
Comment (mukundv-chrome) [reply configs/test/bigquery/datasets.yaml]: It doesn't, I'm pretty sure we deleted it earlier

File: src/appengine/handlers/cron/fuzz_strategy_selection.py (hunk 2 above)
Comment (mukundv-chrome) [reply src/appengine/handlers/cron/fuzz_strategy_selection.py]: Sure.

**Pull request merged**